includes/discovery/ports/nokia-isam.inc.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+// Nokia ISAM has ports that can be accessed in other SNMP context.
+// IHUB contains the ports of the NT cards ans backplane ports from NT to line cards
+SnmpQuery::options($operStatusSnmpFlags)->context('ihub')->hideMib()->walk([
```

Comment:
Prefer not to use the "options" method if possible so we can swap out the snmp implementation.  This seems to be just `-OQUs`, which is not needed.

Reply:
Fixed this in discover.
In poller.inc.php this Options are still needed.